enterprise_access/apps/admin_portal_learner_profile/serializers.py

```diff
@@ -0,0 +1,36 @@
+"""
+Serializers for the admin_portal_learner_profile app.
+"""
+import logging
+
+from rest_framework import serializers
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=abstract-method
+class AdminLearnerProfileRequestSerializer(serializers.Serializer):
+    """Serializer for validating admin learner profile query parameters."""
+    user_email = serializers.EmailField(required=False, help_text="The email address of the learner.")
+    lms_user_id = serializers.IntegerField(required=False, help_text="The ID of the LMS user.")
+    enterprise_customer_uuid = serializers.UUIDField(required=True, help_text="The UUID of the enterprise customer.")
+
+    def validate(self, attrs):
+        """Ensure all required fields are provided."""
+        if not attrs.get('enterprise_customer_uuid'):
+            raise serializers.ValidationError("enterprise_customer_uuid is required.")
+        if not attrs.get('user_email') or not attrs.get('lms_user_id'):
```

Comment:
These fields are marked `required=False` above.  If we're now requiring both of them, should update those Field definitions.